xarray/core/ops.py

```diff
@@ -277,6 +278,15 @@ def count(data, axis=None):
     """
     return sum(~isnull(data), axis=axis)
 
+def rolling_count(rolling):
+
+    not_null = rolling.obj.notnull()
+    instance_attr_dict = {'center': rolling.center,
+                          'min_periods': rolling.min_periods,
+                          rolling.dim: rolling.window}
+    rolling_count = not_null.rolling(**instance_attr_dict).sum()
```

Comment:
I couldn't find a more general / elegant way of doing this, with something like `._shallow_copy`